barretenberg/cpp/src/barretenberg/proof_system/relations/ecc_vm/ecc_lookup_relation.hpp

```diff
@@ -30,6 +30,16 @@ template <typename FF_> class ECCVMLookupRelationBase {
         return (row.msm_add == 1) || (row.msm_skew == 1) || (row.precompute_select == 1);
     }
 
+    template <typename Accumulator, typename AllEntities>
```

Comment:
This was the last component that needed to be "genericized" in order for the `accumulate` logic to be fully generic